kiwi/bootloader/config/grub2.py

```diff
@@ -64,8 +64,8 @@ def post_init(self, custom_args):
         self.firmware = FirmWare(
             self.xml_state
         )
-
-        self.hybrid_boot = True
+        if arch == 'x86_64':
+            self.hybrid_boot = True
```

Comment:
Yes. makes sense, but this is confusing.. 

I see no difference between hybrid  and regular boot entry